src/sentry/models/grouphash.py

```diff
@@ -23,3 +24,38 @@ class Meta:
         app_label = 'sentry'
         db_table = 'sentry_grouphash'
         unique_together = (('project', 'hash'),)
+
+    @staticmethod
+    def fetch_last_processed_event_id(project_id, group_hash_ids):
+        prefix = 'last-processed-event:{}'.format(project_id)
```

Comment:
Long key is long but I'm starting to get worried about key collisions since we don't have any sort of master list about what we're putting at random Redis keys.